docs/whats-new.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: What's new?
+---
+
+At ngrok, we deploy our SaaS platform multiple times a day across our global network. We are constantly improving and adding new features.
```

Comment:
I love this. Thoughts on setting expectations for update frequency and magnitude?

Perhaps:
```
You can expect this page to update at least monthly. We'll include
- feature launches
- major and minor agent, SDK, and API client releases
- dashboard improvements
- account changes
- changes to our API
- major security fixes
```

I think something like that helps us internally as well.